proto/sifnode/oracle/v1/types.proto

```diff
@@ -7,13 +7,47 @@ import "sifnode/oracle/v1/network_descriptor.proto";
 option go_package = "github.com/Sifchain/sifnode/x/oracle/types";
 
 message GenesisState {
-  map<uint32, ValidatorWhiteList> address_whitelist = 1;
+  repeated GenesisValidatorWhiteList validator_whitelist = 1;
   string admin_address = 2;
   repeated Prophecy prophecies = 3;
-  map<uint32, uint32> consensus_needed = 4;
-  map<uint32, CrossChainFeeConfig> cross_chain_fee = 5;
-  map<string, uint64> witness_lock_burn_sequence = 6;
-  map<string, ProphecyInfo> prophecy_info = 7;
```

Comment:
Lets make a message for NetworkConfigData which contains the address_whitelist, consensus_needed, and the cross_chain_fee so we don't have to look them up individually. And these become a repeated struct. This struct also needs to include the NetworkDescriptor number so that indexing can be done properly.

Reply:
will add NetworkConfigData